modules/eco-configuring-control-plane-machine-health-check-with-self-node-remediation.adoc

```diff
@@ -0,0 +1,88 @@
+// Module included in the following assemblies:
+//
+// *nodes/nodes/eco-poison-pill-operator.adoc
+
+:_content-type: PROCEDURE
+[id="configuring-control-plane-machine-health-check-with-self-node-remediation-operator_{context}"]
+= Configuring control-plane machine health checks to use the Self Node Remediation Operator
+
+Use the following procedure to configure the control-plane machine health checks to use the Self Node Remediation Operator as a remediation provider.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Create a `SelfNodeRemediationTemplate` CR:
+
+.. Define the `SelfNodeRemediationTemplate` CR:
++
+[source,yaml]
+----
+apiVersion: self-node-remediation.medik8s.io/v1alpha1
+kind: SelfNodeRemediationTemplate
+metadata:
+  namespace: openshift-machine-api
+  name: selfnoderemediationtemplate-sample
+spec:
+  template:
+    spec:
+      remediationStrategy: ResourceDeletion <1>
+----
+<1> Specifies the remediation strategy. The default strategy is `ResourceDeletion`.
```

Comment:
Since 4.12 we'll have only Resource Deletion strategy, even if you will specify Node Deletion strategy, SNR will use Resource Deletion strategy.


@mshitrit wdyt?

Reply:
I agree, but as far as I can see `Node Deletion` is not mentioned as option (which is good IMO), so I am not sure if any change is required.

Reply:
Thanks @prabinovRedhat @mshitrit 

Regarding this procedure http://file.emea.redhat.com/pogrady/TELCODOCS-502/nodes/nodes/eco-self-node-remediation-operator.html#configuring-machine-health-check-with-self-node-remediation-operator_self-node-remediation-operator-remediate-nodes

'NodeDeletion' is not mentioned so it's ok.

But in this topic http://file.emea.redhat.com/pogrady/TELCODOCS-502/nodes/nodes/eco-self-node-remediation-operator.html#understanding-self-node-remediation-remediation-template-config_self-node-remediation-operator-remediate-nodes

'NodeDeletion' is mentioned with 'Resource Deletion' as two remediation strategies. Should 'NodeDeletion' be deleted in this topic as Polina mentions above - since 4.12 we'll have only the Resource Deletion strategy?

Reply:
I think yes. Since 4.12 (snr 0.5.0) we don't have **self-node-remediation-node-deletion-template**
wdyt @mshitrit